container/container.go

```diff
@@ -12,6 +13,10 @@ import (
 	"github.com/mesg-foundation/core/config"
 )
 
+var (
+	errSwarmNotInit = errors.New(`docker swarm is not initialized. run "docker swarm init" to setup swarm and try again`)
```

Comment:
run `"docker swarm init"` and try again.

Docker swarm init **initialize** swarm. Setup means something diffrent in this case (https://docs.docker.com/engine/swarm/swarm-tutorial/#use-docker-for-mac-or-docker-for-windows)

Also this message is short and clear

Reply:
Done